### PR TITLE
Add get_bytes and set_bytes for bulk storage

### PR DESCRIPTION
**Pull request by armaniferrante**

### CODE REVIEW

File: src/ext.rs [hunk 1]
```diff
@@ -103,9 +103,38 @@ mod external {
 		pub fn input_length() -> u32;
 
 		pub fn fetch_input(dst: *mut u8);
+
+        // Key must be 32 bytes.
```
Comment (peterjgilbert) [inline src/ext.rs]: spaces vs tabs

File: src/ext.rs [hunk 2]
```diff
@@ -103,9 +103,38 @@ mod external {
...
+		pub fn get_bytes(key: *const u8, result: *mut u8);
+        // Key must be 32 bytes.
+        pub fn get_bytes_len(key: *const u8) -> i32;
```
Comment (peterjgilbert) [inline src/ext.rs]: why not u32? or u64?

File: src/ext.rs [hunk 3]
```diff
@@ -103,9 +103,38 @@ mod external {
...
+        // Key must be 32 bytes.
+		pub fn set_bytes(key: *const u8, bytes: *const u8, bytes_len: u64);
 	}
 }
 
+/// Retrieve data directly from the contract storage trie.
+pub fn get_bytes(key: &H256) -> Result<owasm_std::Vec<u8>, Error> {
+    let result_len = get_bytes_len(key)?;
```
Comment (peterjgilbert) [inline src/ext.rs]: spaces vs tabs again

Comment (armaniferrante) [reply src/ext.rs]: Done.

File: src/ext.rs (hunk 1 above)
Comment (armaniferrante) [reply src/ext.rs]: Done.

File: src/ext.rs (hunk 2 above)
Comment (armaniferrante) [reply src/ext.rs]: Changed u64.

**Review by peterjgilbert — APPROVED** [review]

**Pull request merged**